src/ring_theory/tensor_product.lean

```diff
@@ -714,6 +714,46 @@ end
 
 end monoidal
 
-end tensor_product
+section
+
+variables {R A B S: Type*} [comm_semiring R] [semiring A] [semiring B] [comm_semiring S]
+variables [algebra R A] [algebra R B] [algebra R S]
+variables (f : A →ₐ[R] S) (g : B →ₐ[R] S)
+
+variables (S)
+
+/-- The multiplication map on an algebra is a morphism of algebras. -/
```

Comment:
Please include a mention of `algebra.lmul'` in this docstring as I did in my suggestion so that it can be clicked in the web docs.

Reply:
I think it's also worth drawing attention to the fact this requires commutativity of `S`
```suggestion
/-- The multiplication map on a commutative algebra is a morphism of algebras. -/
```